README.md

```diff
@@ -0,0 +1,64 @@
+# Nonlocal Gravity Wave Flux Prediction
+
+The code in this repository implements three classes of deep learning models to predict atmospheric gravity wave momentum fluxes, provided the atmospheric background conditions as input.
+
+## Models description
+The three kinds of models are:
+1. **M1:** A single column ANN which takes single column atmospheric variables as input and predicts the gravity wave momentum fluxes in the single column (pixel-to-pixel regression)
+2. **M2:** A regionally nonlocal CNN+ANN which takes a (2N+1)x(2N+1) horizontal columns of atmospheric variables as input and predicts the fluxes in the central single column (image-to-pixel regression)
+3. **M3:** A global attention UNet model which taked global horizontal maps of variables as input to predict fluxes over the whole horizontal domain (image-to-image regression)
+The three models are schematically depicted in the animation below
+
+
+
+![Schematic for the three models](https://amangupta2.github.io/images/icml_schematic.gif)
+
+
+
+The models are trained on modern reanalysis ERA5 which resolves part of the atmospheric gravity wave spectrum. Since it does not resolve the mesoscale wave spectrum, the repository provides some functionality to retrain parts of the models M1-M3 trained on ERA5 to be retrained on fluxes obtained from a 1 km global IFS model which resolves the whole mesoscale wave spectrum.
+
+
+## Usage
+
```

Comment:
```suggestion
### Requirements
```
It would be useful to have an outline of the requirements here, even if it's just a link to a `requirements.txt`.
Can also note that the code is configured to use cuda devices if available.
Which versions of Python do you want to specify?

Reply:
```suggestion

### Dependencies

To use these scripts, users will first need to install the following dependencies:

* `pytorch`
* `netCDF4`
* `numpy`
* `xarray`

> [!NOTE]
> A `pyproject.toml` or `requirements.txt` will be uploaded shortly, to make installing the dependencies easier.

```